apps/package.json

```diff
@@ -27,7 +27,7 @@
     "build:log": "grunt logBuildTimes",
     "build:stats": "node ./script/build-stats.js",
     "prestorybook": "curl -o build/package/css/application.css http://localhost-studio.code.org:3000/assets/application.css",
-    "storybook": "start-storybook -p 9001 -s build/package/",
+    "storybook": "start-storybook -p 9001 -s build/package/,../dashboard/public",
```

Comment:
Ah nice, does this mean storybook will refresh on (dashboard?) stylesheet changes?